tools/ci/cirrus_arm.yml

```
@@ -86,22 +87,22 @@ freebsd_test_task:
   prepare_env_script: |
     # Create a venv (the `source` command needs bash, not the default sh shell)
     chsh -s /usr/local/bin/bash
-    python -m venv .venv
+    python311 -m venv .venv
```

I think there's a `.` in the name of the binary.
```suggestion
    python3.11 -m venv .venv
```